profiles/mpl_funcs.py

```diff
@@ -0,0 +1,108 @@
+from matplotlib.offsetbox import AnnotationBbox, OffsetImage
+from matplotlib.animation import FuncAnimation
+
+from main_window.competition.comp_app import MAP_MARKER
+from main_window.device_manager import DeviceType
+from main_window.data_entry_id import DataEntryIds
+from main_window.main_app import MainApp
+from textwrap import wrap
+
+from main_window.mplwidget import MplWidget
+
+label_to_dataID = {"Altitude":DataEntryIds.CALCULATED_ALTITUDE,
+                   "MaxAltitude":None,
+                   "State":DataEntryIds.STATE,
+                   "Stage2State":DataEntryIds.STATE,
+                   "Pressure":DataEntryIds.PRESSURE,
+                   "Acceleration": DataEntryIds.ACCELERATION_X,
```

Comment:
Perhaps add acceleration in the Y and Z directions as well

Reply:
<img width="600" alt="Screen Shot 2022-05-16 at 11 03 13 PM" src="https://user-images.githubusercontent.com/77385101/168740071-5e512846-12cc-4092-aba1-31b9dfd18c98.png">

What do you think about putting Y,Z acceleration on the same graph?
Alternatively, I can add a net acceleration column in the rocket data and plot that. 

I changed all scatter plot graphs to connected lines. Hopefully easier to read?

Reply:
Having them all on the same graph is nice, good idea! Can we also have a checkbox list so the user can choose which out of the 3 are plotted at the same time?

Reply:
<img width="1047" alt="Screen Shot 2022-06-01 at 11 27 12 PM" src="https://user-images.githubusercontent.com/77385101/171520068-8b089dfb-885d-41c4-afe8-5a79fc1922f0.png">